test/runtests.jl

```diff
@@ -51,6 +52,46 @@ Random.seed!(100)
     @test cost2 ≈ cost
 end
 
+@testset "1D Optimal Transport for Convex Cost" begin
+    # Continuous Case
+    μ = Normal(0, 2)
+    ν = Normal(10, 2)
+    c(x, y) = abs(x - y)
+    γ = optimal_transport_plan(c, μ, ν)
+
+    @test optimal_transport_cost(c, μ, ν) ≈ 10 atol = 1e-5
+    @test optimal_transport_cost(c, μ, ν; plan=γ) ≈ 10 atol = 1e-5
+
+    # Discrete Case
+    n, m = 100, 150
+
+    μ = rand(n)
+    ν = rand(m) .+ 0.5
+    μ_n = rand(n)
+    ν_m = rand(m)
+    μ_n = μ_n / sum(μ_n)
+    ν_m = ν_m / sum(ν_m)
+
+    μ = DiscreteNonParametric(μ, μ_n)
+    ν = DiscreteNonParametric(ν, ν_m)
+
+    c(x, y) = (x - y)^2
+    # new version of StatsBase also has functoin pairwise,
+    # which conflicts with Distances.pairwise
+    C = Distances.pairwise(SqEuclidean(), μ.support, ν.support)
```

Comment:
No, there should not be any conflicts. They both just add methods to `StatsAPI.pairwise` and reexport it, it's exactly the same function.
```suggestion
    C = pairwise(SqEuclidean(), μ.support, ν.support)
```